.storybook/preview-head.html

```diff
@@ -0,0 +1,11 @@
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100..900;1,100..900&display=swap"
+  rel="stylesheet">
+<style>
+  body {
+    font-family: 'Work Sans', sans-serif;
+    font-optical-sizing: auto;
+    font-style: normal;
+  }
+</style>
```

Comment:
Better to have `preview.css` file in this folder and import the CSS file here.

Reply:
Done